llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -46,11 +46,13 @@ struct RISCVTuneInfo {
 
 class RISCVSubtarget : public RISCVGenSubtargetInfo {
 public:
+  // clang-format off
   enum RISCVProcFamilyEnum : uint8_t {
     Others,
     SiFive7,
+    VentanaVeyron
```

Comment:
Leave a trailing comma after this. Makes it easier to insert new CPUs without touching this line.

Reply:
done